tests/pipeline/test_comoposite_planner.py

```diff
@@ -0,0 +1,294 @@
+import pytest
```

Comment:
The filename here is spelled wrong. `test_composite_planner.py`

Reply:
I promise I will buy a new keyboard during the thanksgiving weekend